mod_modular_vanilla/hooks/entity/tactical/actor.nut

```diff
@@ -443,20 +445,20 @@
 				{
 					if (damage > 0 && !this.isHiddenToPlayer())
 					{
-						this.Tactical.EventLog.logEx(this.Const.UI.getColorizedEntityName(this) + "\'s " + this.Const.Strings.BodyPartName[_hitInfo.BodyPart] + " is hit for [b]" + this.Math.floor(damage) + "[/b] damage");
+						::Tactical.EventLog.logEx(format("%s\'s %s (%i) is hit for %i damage", ::Const.UI.getColorizedEntityName(this), ::Const.Strings.BodyPartName[_hitInfo.BodyPart], oldHitpoints, ::Math.floor(damage)));
```

Comment:
As a general rule we shouldn't change vanilla behavior i.e. when someone installs Modular Vanilla into vanilla BB he shouldn't observe anything different in the game.

We can modularize these functions and implement ways for others to change what is printed to the log.